vault/external_tests/raftha/raft_ha_test.go

```diff
@@ -364,6 +364,9 @@ func TestRaftHACluster_Removed_ReAdd(t *testing.T) {
 			if !server.Healthy {
 				return fmt.Errorf("server %s is unhealthy", serverID)
 			}
+			if server.NodeType != "voter" {
```

Comment:
this isn't related to the PR, but I wanted to fix the race test flake. I ran locally 5 times and didn't see it fail, when previously it would fail 50% of the time locally